components/stepper/overview.md

```diff
@@ -16,25 +16,20 @@ The <a href ="https://www.telerik.com/blazor-ui/stepper"><strong>Stepper for Bla
 The Stepper provides several features out of the box, including the ability to define the appearance of each [step]({%slug stepper-steps-overview%}), [validation]({%slug stepper-steps-validation%}) for the steps, strict [linear flow]({%slug stepper-linear-flow%}) and [step template]({%slug step-template%}) to make the it fit any design requirements.
 
 
-#### In this article:
-   * [Basics](#basics)
-   * [Features](#features)
+## Creating Blazor Stepper
 
-## Basics
+1. Add the `TelerikStepper` tag to add the component to your razor page.
 
-To use a Telerik Stepper for Blazor:
+2. Set the `Value` parameter. It supports one-way and two-way binding.
 
-1. Use the `TelerikStepper` tag
+3. Configure the [`StepperStep` instances]({%slug  stepper-steps-overview%}) inside the `StepperSteps` tag.
```

Comment:
Configure what? If Label and Icon are optional, what is the minimum required configuration that we can mention here?